README.md

```diff
@@ -1,16 +1,33 @@
 # mbed-client-cli
 
-This is the Command Line Library for a CLI application. This library provides methods for:
+This is the Command Line Library for a CLI application. It uses only ansi C features so it is portable and works in mbed-os-3, mbed-os-5, linux and windows.
+
+## Features
+
+Library provides features such:
 
 * Adding commands to the interpreter.
 * Deleting commands from the interpreter.
 * Executing commands.
 * Adding command aliases to the interpreter.
 * Searching command arguments.
+* implements several VT100/VT220 features, e.g.
+  * move cursor left/right (or skipping word by pressing alt+up/right)
+  * delete characters
+  * CTRL+W to remove previous word
+  * browse command history by pressing up/down
+* implements basic commands, e.g.
+  * echo
+  * help
+  * (un)set
```

Comment:
the help message has no `(un)set` only `set`

Reply:
I did not add any title/mans for these commands to save some memory.. maybe there could be some short text..

Reply:
it should be there - maybe these changes was not included in your build. See:
```
/>help 
Commands:
help            This help
echo            Echo controlling
alias           Handle aliases
unset           unset variables
set             print or set variables
clear           Clears the display
history         View your command Line History
true            
false           
exit            exit shell
dummy           dummy command
```

Reply:
okay, I see. I was testing with mbed-os-5-example.bin build by CI, but the mbed-client-cli.lib isn't up to date.

Reply:
linux example `./cli`
1.  command being echoed twice.  
2 .VT100/220 features didn't work, except for `ctrl+w`
```
/>set --retcode 1
/>set --retcode 1 
retcode: 0
/>true
/>true 
retcode: 0
```